packages/esm-service-queues-app/src/queue-table/default-queue-table.test.tsx

```diff
@@ -106,17 +106,17 @@ describe('ActiveVisitsTable: ', () => {
     expect(john).toBeInTheDocument();
     expect(john).toHaveAttribute('href', 'someUrl');
 
-    const expectedColumnHeaders = [/name/, /priority/, /status/, /waitTime/];
+    const expectedColumnHeaders = ['Name', 'Priority', 'Coming from', 'Status', 'Queue', 'Wait time', 'Actions'];
```

Comment:
nit: Regex matches might be preferable to avoid getting burned by changes in case.